docs/proposals/proposal-workflow-authentication.md

```diff
@@ -0,0 +1,69 @@
+# Proposal for Workflow Authentication
+
+## Proposal summary
+
+**For workflow-to-workflow-service authentication, use:**
+
+* **for jobs: one-time per-job tokens**, i.e. a single one-time token for
+  every job;
+* **for the user/interactive CLI: timed tokens**.
```

Comment:
For the same logged in user with full access to the file system where the WFS is running as the same user.

Out of scope: Future CLI commands that are not invoked under the above condition may go through the UIS.